public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -36,11 +31,7 @@ class AtomEdit extends React.Component {
     this.props.atomActions.updateAtom(newAtom);
   }
 
-  render () {
-
-    if (!this.props.atom) {
-      return <div>Loading...</div>;
-    }
+  renderSpecificEditor () {
 
     //TODO: This is brittle, can we improve?
     const atomType = this.props.atom.atomType.toLowerCase();
```

Comment:
the object immediately inside `data` has the same key as the type of the atom. Not the nicest but it'll get around the use of `toLowerCase`